src/styles/_variables.scss

```diff
@@ -1,19 +1,23 @@
-// colors defined by BDL
+// Box Design Language Theme Colors
 $bdl-box-blue: #0061d5 !default;
```

Comment:
Should these blues live under `// Box Design Langauge Blues`? 🎶

Reply:
Actually we should remove the "blues" section until it's decided whether the scale will have 3 or 5 colors